command/sub8_launch/launch/tf.launch

```diff
@@ -29,6 +29,10 @@
 
     <node pkg="sub8_launch" type="tf_republisher.py" name="tf_replublisher" />
 
+    <node pkg="tf" type="static_transform_publisher" name="blueview_broadcaster"
+      args="0.333 0 0.381  0 0 0 1 base_link blueview 100">
```

Comment:
It is, but the data it produces treats X, Y, and Z as Daniel described. So having the correct transform here would be nice so the ogrid/pointcloud published from the data will transform correctly.